model-optimizer/mo/utils/versions_checker.py

```diff
@@ -175,7 +175,7 @@ def version_check(name, installed_v, required_v, sign, not_satisfied_v):
         not_satisfied_v.append((name, 'installed: {}'.format(installed_v), 'required: {} {}'.format(sign, required_v)))
 
 
-def get_environment_setup():
+def get_environment_setup(framework):
     """
     Get environment setup such as Python version, TensorFlow version
     :return: a dictionary of environment variables
```

Comment:
Update the function description with information about a new function parameter `framework`